test/NameBinding/accessibility.swift

```diff
@@ -63,6 +63,17 @@ _ = { rdar27982012($0.0) }((1, 2)) // expected-error {{initializer is inaccessib
 _ = PrivateInit() // expected-error {{'PrivateInit' initializer is inaccessible due to 'private' protection level}}
 // TESTABLE: :[[@LINE-1]]:{{[^:]+}}: error: 'PrivateInit' initializer is inaccessible due to 'private' protection level
 
+_ = OtherModuleMemberwiseInit(x: 42) // expected-error {{memberwise initializer of 'OtherModuleMemberwiseInit' is inaccessible due to 'internal' protection level; public memberwise initializers must be declared explicitly}}
+
+public struct PrivateMemberwiseInit {
```

Comment:
I wonder if would be good to have an extra test case, since the note is emitted for all members that contribute to access level. Maybe can be good add a test case where it has more than one member :)